Automattic/woocommerce-payments
# Make MC Settings screen to use a 2 column layout

Fixes #2745 

#### Changes proposed in this Pull Request

Make MC Settings screen to use a 2 column layout

MC settings need to match the new design of the WC Payments settings screen, using the same 2 column layour.

In order to do that, I've moved some parts of the MC Settings screen rendered by PHP to React. The titles and descriptions are all now in React. The Store Settings section checkboxes are still rendered by PHP but position accordingly inside of the SettingsSection for achieving the desired layout. 

Before:

![image](https://user-images.githubusercontent.com/2612426/131835478-4810a86f-a65c-4f20-81cd-a79bf17ed3f0.png)

After:

![image](https://user-images.githubusercontent.com/2612426/133769450-0fc2f94c-e979-4180-911a-cd25a9aaffcd.png)

#### Testing instructions

* Go to Payments - Settings - Multi-currency
* See new design
* See that the settings are still functional

-------------------


**Post merge**
